docs/docs-content/security-bulletins/cve-reports.md

```diff
@@ -32,6 +32,35 @@ _Are there any links users can visit to find out more?_
 
 -->
 
+## April 2, 2024 - CVE-2024-3094 Malicious code was discovered in the upstream tarballs of xz - 10 CVSS
+
+Malicious code was discovered in the upstream tarballs of xz, starting with version 5.6.0. Through a series of complex 
+obfuscations, the liblzma build process extracts a prebuilt object file from a disguised test file existing in the 
+source code, which is then used to modify specific functions in the liblzma code. 
+This results in a modified liblzma library that can be used by any software linked against this library, intercepting 
+and modifying the data interaction with this library.
+
+#### Impact
+
+No impact. Palette uses Ubuntu 22.04 OS images while VerteX uses Ubuntu 20.04 OS images for Enterprise, 
```

Comment:
Does this refer to self-hosted Palette and VerteX? What about non-hosted environments, such as workload clusters?

Is this also true for someone running version 3.4?

Reply:
Tenant cluster is mentioned in that statement. We can change it to workload cluster if that's how we are referring them.
Yes, it is true for all versions supported.